website/content/en/docs/olm-integration/testing-deployment.md

```diff
@@ -8,51 +8,120 @@ This document discusses the behavior of `operator-sdk <run|cleanup>` subcommands
 and assumes you are familiar with [OLM][olm], related terminology,
 and have read the SDK-OLM integration [design proposal][sdk-olm-design].
 
-Currently only the package manifests format is supported by `<run|cleanup>` subcommands. Bundle support is coming soon.
-
 **Note:** before continuing, please read the [caveats](#caveats) section below.
 
-## `operator-sdk <run|cleanup> packagemanifests` command overview
+## `operator-sdk run bundle` command overview
+`operator-sdk run bundle` assumes OLM is already installed and running on your
+cluster. It also assumes that your Operator has a valid [bundle][bundle-format].
+See the [creating a bundle][creating-bundle] guide for more information. See the
+[CLI overview][doc-cli-overview] for commands to work with an OLM installation
+and generate a bundle.
+
+Let's look at the configuration shared between `run bundle`, `run
+packagemanifests` and `cleanup`:
+
+- **kubeconfig**: the local path to a kubeconfig. This uses well-defined default
+  loading rules to load the config if empty.
+- **namespace**: the cluster namespace in which Operator resources are created.
+  This namespace must already exist in the cluster.
+- **timeout**: a time string dictating the maximum time that `run` can run. The
+  command will return an error if the timeout is exceeded.
+
+Let's look at the anatomy of the `run bundle` configuration model:
+
+- **bundle-image**: specifies the Operator bundle image, this is a
+  required parameter. The bundle image must be pullable.
+- **index-image**: specifies an index image in which to inject the given bundle.
+  This is an optional field which will default to
+  `quay.io/operator-framework/upstream-opm-builder:latest`
+- **install-mode**: specifies which supported [`installMode`][csv-install-modes]
+  should be used to create an `OperatorGroup` by configuring its
+  `spec.targetNamespaces` field. The `InstallModeType` string passed must be
+  marked as "supported" in the CSV being installed.
+  - This option understands the following strings (assuming your CSV does as
+    well):
+    - `AllNamespaces`: the Operator will watch all namespaces (cluster-scoped
+      Operators). This is the default.
+    - `OwnNamespace`: the Operator will watch its own namespace (from
+      **namespace** or the kubeconfig default).
+    - `SingleNamespace="my-ns"`: the Operator will watch a namespace, not
+      necessarily its own.
+
```

Comment:
If the `CSV` does not support either of those, then am I right that `--install-mode` flag will be required?

If that's the case, we should document that as well.

Reply:
Added clarification.